pkg/expression/util.go

```diff
@@ -1556,6 +1556,12 @@ func MaybeOverOptimized4PlanCache(ctx BuildContext, exprs []Expression) bool {
 	return containMutableConst(ctx.GetEvalCtx(), exprs)
 }
 
+// MaybeOverOptimized4PlanCacheForMultiExpression is the same as MaybeOverOptimized4PlanCache,
+// but it accepts multiple expressions as input.
+func MaybeOverOptimized4PlanCacheForMultiExpression(ctx BuildContext, exprs ...Expression) bool {
```

Comment:
```MaybeOverOptimized4PlanCacheForMultiExpression``` can replace `MaybeOverOptimized4PlanCache`. But this PR is for bugfix. I don't want to change too many.